src/librustc/diagnostics.rs

```diff
@@ -1950,9 +1950,9 @@ const E: &'static Cell<usize> = &D.a; // error
 const F: &'static C = &D; // error
 ```
 
-This is because cell types internally use `UnsafeCell`, which isn't `Sync`.
-These aren't thread safe, and thus can't be placed in statics. In this case,
-`StaticMutex` would work just fine, but it isn't stable yet:
+This is because cell types do operations that are not thread-safe, and
+hence do not implement `Sync`. Thus can't be placed in statics. In this
```

Comment:
"Thus can't be placed in statics" isn't a sentence.

This is because cell types do operations that are not thread-safe. Due to this they don't implement `Sync` and thus can't be placed in statics.